syfertext/utils.py

```diff
@@ -123,8 +124,7 @@ def _download_model(model_name: str, model_path: str):
                     prog_bar.update(chunk_size)
 
                 else:
-
                     # Downloading the file finished
                     break
-
     prog_bar.close()
+    return tmp_model
```

Comment:
Please also add a test in `tests/local_mode/test_language`

You can call it `test_language_model_download_path()`

We you load the language model with

```
syfertext.load('en_core_web_lg', owner = me)
```

Then you make sure that the model folder is in the expected directly